README.md

```diff
@@ -127,6 +134,19 @@ Global Flags:
 
 **NOTE**: The `score-compose run` command still exists but is hidden and should be considered deprecated as it does not support resource provisioning.
 
+### Using the `--publish` flag
```

Comment:
Should we mention here (or in the flag description above) that this directive should be the last one of the command?

Reply:
Made it more clear in the docs and clarified some things :)